### ISSUE TITLE
System updater misleading words

### ISSUE THREAD

**Issue by matjazpotocnik** [processwire/processwire-issues#1064]
After system update 17 is performed (or you manually trigger an update in the System Updater module settings), I'm informed that my system is not apache and I should perform manual update by following the details in the URL below. See the images:

![System updater 1](https://user-images.githubusercontent.com/7663992/72364402-c00aa800-36f6-11ea-8153-72d564ce9ea0.png)

![System updater 2](https://user-images.githubusercontent.com/7663992/72364414-c567f280-36f6-11ea-9441-406e8c3d8bc2.png)

But the details are above this message, not below.

On the other hand, in AdminThemeDefault it looks like this (ok):

![System updater 3](https://user-images.githubusercontent.com/7663992/72364961-aae24900-36f7-11ea-8596-9409478bd12b.png)

**Comment by matjazpotocnik** [comment]
I also noticed that the order of messages changes. I Tried this in admin.php:

```
$this->error('E1');
$this->error('E2');

$this->message('M1', Notice::noGroup);
$this->message('M2', Notice::noGroup);

$this->warning('W1', Notice::noGroup);
$this->warning('W2', Notice::noGroup);
```

And this is how it shows up when Submit is clicked or the page is reloaded (please open the video on new tab):

![notices](https://user-images.githubusercontent.com/7663992/72406040-393be680-375b-11ea-910d-e030b79ac3ff.gif)

Observe the order of notices. On submitting the order is "message, error, warning, message", while on page reload, the order is "error, message, warning", that's the order specified in admin.php. Things change when you use AdminThemeDefault: on submitting the form it's "message, error, warning" (and all messages are grouped), on page reload it's "error, message, warning" (consistent with AdminThemeUikit).

**Comment by ryancramerdesign** [comment]
Thanks @matjazpotocnik I have pushed a fix. The error was that the update was using language like "see below" when it shouldn't have, so I have updated it to link the URL directly. 

The reason for the difference in admin themes there is that AdminThemeUikit supports grouping the notifications (and does so by default) whereas the older admin themes do not. 

As for whether the messages, warnings or errors group comes first, there isn't a specific intention here, and it just uses whatever type is called first. But when notices get stuffed into the session for display on the next request (like a POST request that redirects after saving), then they always go in the order of: messages, errors, warnings — because that's the order they are stored in by $session. The relative order of messages to messages, errors to errors, etc., remains. I'm not sure it matters, but it could always store them in a different group order, like errors, warnings, messages. This part is not affected by admin theme.
